pkg/types/tool.go

```diff
@@ -29,15 +29,15 @@ var (
 type ToolType string
 
 const (
-	ToolTypeContext    = ToolType("context")
-	ToolTypeAgent      = ToolType("agent")
-	ToolTypeOutput     = ToolType("output")
-	ToolTypeInput      = ToolType("input")
-	ToolTypeAssistant  = ToolType("assistant")
+	ToolTypeContext = ToolType("context")
+	ToolTypeAgent   = ToolType("agent")
+	ToolTypeOutput  = ToolType("output")
+	ToolTypeInput   = ToolType("input")
+	//ToolTypeAssistant  = ToolType("assistant")
 	ToolTypeTool       = ToolType("tool")
 	ToolTypeCredential = ToolType("credential")
-	ToolTypeProvider   = ToolType("provider")
-	ToolTypeDefault    = ToolType("")
+	//ToolTypeProvider   = ToolType("provider")
```

Comment:
Did you mean to delete these entirely?

Reply:
Oh, I'll clean that up so it makes sense.